QUEST_LV_0200.tsv

```diff
@@ -3012,63 +3012,63 @@ QUEST_LV_0200_20150317_003011	$Defeat Saltistter that attacked Vacenin Squad
+QUEST_LV_0200_20150317_003038	$You retrieved the Holy Relic of Salvation. Talk with Bishop Aurelius.
+QUEST_LV_0200_20150317_003039	$Retrieve %s by defeating Loftlems
+QUEST_LV_0200_20150317_003040	$Retrieve the Holy Relic of Salvation by defeating Loftlems
 QUEST_LV_0200_20150317_003041	{memo X}You found all divine artifacts. Talk with Aurelius to get the first key of Maven.
 QUEST_LV_0200_20150317_003042	{memo X}Put the divine artifact into the altar of sanctuary
 QUEST_LV_0200_20150317_003043	{memo X}Put the appropriate divine artifact into the altar of sanctuary to obtain the key. If you have a trouble finding the key, call Bishop Aurelius to get some help.
 QUEST_LV_0200_20150317_003044	{memo X}You've acquired the first key of Maven. Talk with Bishop Aurelius.
-QUEST_LV_0200_20150317_003045	Talk with the disciple, Inea
-QUEST_LV_0200_20150317_003046	The disciple, Inea seems to be very mad. Talk with the disciple, Inea.
-QUEST_LV_0200_20150317_003047	Look for the divine artifact of the words
-QUEST_LV_0200_20150317_003048	The disciple, Inea told you that the congregation ordered to retrieve all the divine artifacts in the Great Cathedral. Retrieve all the divine artifacts of the words that are scattered on the floor of the Great Cathedral.
-QUEST_LV_0200_20150317_003049	Hand it over to the disciple Inea.
-QUEST_LV_0200_20150317_003050	You found all the divine artifacts of the words. Hand them over to the disciple, Inea.
-QUEST_LV_0200_20150317_003051	Talk with the disciple, Inea.
-QUEST_LV_0200_20150317_003052	The disciple Inea seems to have some trouble looking for divine artifacts. Talk with the disciple, Inea.
-QUEST_LV_0200_20150317_003053	Find the divine artifacts using the search marble
-QUEST_LV_0200_20150317_003054	Some divine objects disappear themselves when they detect vicious energies. Use the search marble to find the hidden divine objects.
-QUEST_LV_0200_20150317_003055	You've retrieved all the divine objects of the blessing. Hand them over to the disciple, Inea.
-QUEST_LV_0200_20150317_003056	Talk with Disciple Aden
-QUEST_LV_0200_20150317_003057	The disciple Aden seems to be in a trouble. Talk with Disciple Aden.
-QUEST_LV_0200_20150317_003058	Bring the research reports of the disciples
-QUEST_LV_0200_20150317_003059	The disciple, Aden is investigating how to release the curse of Naktis so he wants to look at the reports of other disciples. Obtain the reports from Yosana at Grand Corridor, the disciple Gadan at Penitence Route of Cathedral, and from Prosit at Sanctuary.
+QUEST_LV_0200_20150317_003045	$Talk to Priest Inea
+QUEST_LV_0200_20150317_003046	$Priest Inea seems to be irritated. Talk with Priest Inea.
+QUEST_LV_0200_20150317_003047	$Look for the Scriptural Relics
+QUEST_LV_0200_20150317_003048	$Priest Inea told you that the congregation ordered to retrieve all the Relics in the Great Cathedral. Retrieve all the Scriptural Relics that are scattered on the floor of the Great Cathedral.
```

Comment:
Relics -> relics (to not clutter your PR with more notes, check this for other instances if you do this, **only in dialogue from the NPC**)